omero/sysadmins/public.rst

```diff
@@ -138,3 +139,133 @@ authentication. For example a direct link to image will look as follows::
 For more details about how to create an OMERO session see
 :doc:`server-side session </developers/Server/Sessions>` or
 use the :doc:`command line interface </users/cli/sessions>` to create one.
+
+.. _hosting_data_example:
+
+Complete workflow for hosting data for a publication
+----------------------------------------------------
+
+Putting the pieces of this puzzle together, the following describes the steps
+of a complete workflow for using OMERO to host public data associated with a
+publication. It is illustrated using an example publication from the Swedlow
+lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_ with the
+data hosted at
+`<https://omero.lifesci.dundee.ac.uk/pub/schleicher-et-al-2017>`_.
+
+Ansible playbooks can be found describing how the production server in Dundee
+("nightshade") was configured in the
+`prod-playbooks <https://github.com/openmicroscopy/prod-playbooks>`_
+repository on GitHub.
+
+Group setup
+^^^^^^^^^^^
+
+A group-per-publication allows the public user to be selected added (or
+removed) from given publications to decide their visibility.
+
+#. Create a dedicated read-only group to host the raw data underlying the
+   publication (see :doc:`cli/usergroup`).
+#. Add all the authors of the paper to this new group.
+#. Once you have configured OMERO.web to create a public user (see below), add
+   the public user as a member of the newly created read-only group.
+
+Configuring OMERO.web
+^^^^^^^^^^^^^^^^^^^^^
+
+Either your main OMERO.web server or an additional publication-only server
+will need to be specially configured to permit loginless access. This example
+uses a second, dedicated server.
+
+#. Configure the :ref:`public_user`.
+#. Configure the OMERO.web :ref:`filter on the public user <public.url_filter>` by setting
+   :property:`omero.web.public.url_filter` to allow 'webclient' so that the
+   full webclient is visible for the public user, and thus the Data tree with
+   Projects and Datasets is also browsable, as well as the Tags tab and the
+   full image viewer.
+#. In order to have an automatically logged-in public user while still giving
+   your existing OMERO users an unchanged user experience (i.e. not
+   automatically logging them in as the public user), a dedicated,
+   :doc:`separate web server <unix/install-web/web-deployment>` for servicing
+   the public workflows can be added and configured to point at your existing
+   OMERO.server (in this case, the public user and the public URL filter need
+   to be set on this separate OMERO.web installation).
```

Comment:
It might be better to discuss the logic / decision-making for setting up a second web server as a first step *before* the config steps. Currently it looks like you can make this decision at point 3, after doing steps 1 and 2.

E.g.
"Configuring OMERO.web

If you wish to have an automatically logged-in public user while still giving your existing OMERO users an unchanged user experience (i.e. not automatically logging them in as the public user), a dedicated, separate web server for servicing the public workflows can be added and configured to point at your existing OMERO.server. 
This is the workflow we adopted by adding a public OMERO.web at omero.lifesci.dundee.ac.uk, without changing the existing OMERO.web at nightshade.openmicroscopy.org.

#. Create the public user as described above. 
#. Configure the public user settings for OMERO.web, setting the :property:`omero.web.public.url_filter` to allow access to the full webclient UI as :ref:`described <public.url_filter>`.

Reply:
Pushed